packages/common/src/client/appeals-api-client.js

```diff
@@ -164,10 +165,97 @@ class AppealsApiClient {
 		return response.json();
 	}
 
+	/**
+	 * @param {string} caseReference
+	 * @returns {Promise<(LPAQuestionnaireSubmission)>}
+	 */
+	async getLPAQuestionnaire(caseReference) {
+		const endpoint = `${v2}/appeal-cases/${caseReference}/lpa-questionnaire-submission`;
+		const response = await this.#makeGetRequest(endpoint);
+		return response.json();
+	}
+
+	/**
+	 * @param {string} caseReference
+	 * @param {string} lpaCode
+	 * @returns {Promise<(LPAQuestionnaireSubmission)>}
+	 */
+	async postLPAQuestionnaire(caseReference, lpaCode) {
+		const urlParams = new URLSearchParams();
+		urlParams.append('lpa-code', lpaCode);
+		const endpoint = `${v2}/appeal-cases/${caseReference}/lpa-questionnaire-submission?${urlParams.toString()}`;
```

Comment:
If lpa-code is part of this API, it's not in the spec. Is this needed? There is only ever one submission per case.

Reply:
Following the refactoring and understanding how to use prisma client a bit more I've taken this out